### DISCUSSION TITLE
3dm and xml files about custom robots

### DISCUSSION THREAD

**Discussion by JunJie-zhang-o** [visose/Robots#254] [Q&A] [answered]
I found that the parameters of the same brand and series of multi-model robots are described in the xml file, but there is only one robot model in 3dm. Excuse me, if I want to customize the robot, which model of robot should I make?

**Comment by visose** [comment, marked as answer]
No, you can add multiple robot models on the same 3dm file. See for example the Bartlett library, pay attention to the names of the top level layers.
